archived/star-rx72n-firmware/tests/mocks/mock_usb0_regs.c

```
@@ -86,9 +104,29 @@ void mock_usb0_set_fifo_dtln(uint16_t len)
 
 void mock_usb0_set_pipe1_busy(uint8_t busy)
 {
+  mock_usb0_set_pipe_busy(1, busy);
+}
+
+void mock_usb0_set_pipe_busy(uint8_t pipe, uint8_t busy)
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'mock_usb0_set_pipe_busy' of similar type ('int') are easily swapped by mistake

(bugprone-easily-swappable-parameters)